research/CongressKeynote/Congress-Keynote.md

```diff
@@ -0,0 +1,466 @@
+{% include figure.html src="Slides/Slide37.png" caption="Slide 37." %}
+
+This slide displays a 12-hour night-time false-colour spectrogram that includes the chorusing activity of three frog species (see labels). 
+But you should also be aware of the soundmarks in this image.
+
+* Can you identify, the morning and evening chorus?
+
+* What is that pink line at 4500 Hz?
+
+* Why is the pink line at 7500 Hz descending slightly?
+
+The take-home message is that the starts and ends of chorusing behaviour can be clearly identified using combinations of acoustic indices in LDFC spectrograms. 
+
+---
+
+## Slide 35: Detecting cane-toads
+{% include figure.html src="Slides/Slide38.png" caption="Slide 38." %}
+
+Likewise, the long-duration calling of the cane toad can be easily detected in LDFC spectrograms. At the bottom of this LDFC spectrogram is the output of a custom built cane toad recogniser. It has clearly picked up the cane toad - but it was a lot more work to write the custom recogniser than to use the generic features of the LDFC spectrogram.     
+
+---
+
+## Slide 36: Detecting cryptic species
+{% include figure.html src="Slides/Slide39.png" caption="Slide 39." %}
+
+Perhaps the biggest surprise we have had in the QUT Ecoacoustics Group recently is the usefulness of generic acoustic indies in detecting cryptic and/or threatened bird species such as the Lewin's Rail, Black Rail, Least Bittern and Bhutanese White Heron.
+
+The advantages of this technique are:
+1: Compressed timescale; 2: Generic features; 3: get soundscape + species detection
+
+- Faster preparation of datasets because of labelling at low resolution low resolution, i.e. one label per one minute.
+
+- Faster preparation of recognisers because of using generic indices.
+
+- Get both soundscape visualisation and species recogniser in the same processing steps.
+
+- Visual scanning of the LDFC spectrograms allows for fast checking of false-positive and negatives and has led to the discovery of new species for a given location.  
+
+These advantages come with a serious limitation – the species of interest is only detected where its calls are the dominant acoustic activity in its frequency band over the minute. Surprisingly, the technique has proved to be extremely useful, despite this limitation.
+
+---
+
+## Slide 37: Detecting bats
+Given the low resolution of the LDFC technique as originally developed we had no expectation that would prove useful for identification of echolocating micro-bats. However, we noticed the "red/orange" events in the late evening of several days in March 2016 (see the below stacked ribbon plot) and eventually identified the source as the *white-striped, free-tailed bat*.
+{% include figure.html src="Slides/Slide41.png" caption="Slide 41." %}
+
+---
+
+## Slide 38: Detecting bats
```

Comment:
"call comes below the Nyquist" sounds unusual what about "call is below the Nyquist"